tests/by-util/test_fmt.rs

```diff
@@ -159,3 +185,87 @@ fn test_fmt_set_goal_not_contain_width() {
             .stdout_is("this is a file with one word per line\n");
     }
 }
+
+#[test]
+fn split_does_not_reflow() {
+    for arg in ["-s", "-ss", "--split-only"] {
+        let (at, mut ucmd) = at_and_ucmd!();
+        ucmd.arg("one-word-per-line.txt")
+            .arg(arg)
+            .succeeds()
+            .stdout_is(at.read("one-word-per-line.txt"));
+    }
+}
+
+#[test]
+fn prefix_minus() {
+    for invocation in [
+        vec!["-p-", "prefixed-one-word-per-line.txt"],
+        vec!["-p", "-", "prefixed-one-word-per-line.txt"],
+        vec!["--prefix=-", "prefixed-one-word-per-line.txt"],
+        vec!["--prefix", "-", "prefixed-one-word-per-line.txt"],
+        vec!["--pref=-", "prefixed-one-word-per-line.txt"],
+        vec!["--pref", "-", "prefixed-one-word-per-line.txt"],
+        // Test self-overriding:
+        vec!["--prefix==", "--prefix=-", "prefixed-one-word-per-line.txt"],
```

Comment:
To eliminate the duplication of the filename, I suggest to move it to an `arg` call to make it obvious that it's always the same filename: 
```rust
ucmd.args(&invocation)
    .arg("prefixed-one-word-per-line.txt")
    .succeeds()
    .stdout_is(at.read("prefixed-one-word-per-line_p-.txt"));
```
The same applies to the tests below.

Reply:
Oh! So obvious, yet I didn't notice it. Yes, thanks!